### PR TITLE
Shortcode for total # of visitors (e.g. a site counter)

### PR DESCRIPTION
**Pull request by intentionally-left-nil**
This PR provides the ability to add the total # of visitors as a shortcode.
You can use it in your site (say in footer.php) with the following:
```html
<span>
  Visitors:
  <?php
    echo do_shortcode('[koko_analytics_site_counter days=30]');
  ?>
</span>
```

If there's support for the PR I'm happy to clean it up, provide unit tests, support for pageviews vs visitors, etc.

Thanks!

### CODE REVIEW

File: koko-analytics.php [hunk 1]
```diff
@@ -68,6 +68,10 @@
 $shortcode = new Shortcode_Most_Viewed_Posts();
 $shortcode->init();
 
+require __DIR__ . '/src/class-shortcode-site-counter.php';
+$site_counter_shortocde = new ShortCode_Site_Counter();
+$site_counter_shortocde->init();
```
Comment (AnilRedshift) [inline koko-analytics.php:73]: oops, typo

**Comment by AnilRedshift** [comment]
Hey @dannyvankooten, this PR has been sitting for awhile. Do you mind taking a look and giving feedback?

**Comment by dannyvankooten** [comment]
Hey @intentionally-left-nil,

I'm so sorry for not seeing this before and not getting back until now. I really like the functionality you added here and hope to be able to take a closer look and merge this real soon. The final naming of things might differ a bit, but rest assured that the functionality will end up in the core plugin!

**Comment by dannyvankooten** [comment]
It's finally merged @intentionally-left-nil! Thank you for being so patient.

**Comment by intentionally-left-nil** [comment]
Sweet!

**Pull request merged**